.wuf/update-components.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+rm ./.wuf/components.json
```

Comment:
sorry for nitpicking a bit more :)

```suggestion
rm .wuf/components.json || true
```

1. the `./` is not necessary at the beginning of path
2. the `|| true` part will allow `rm` to fail but continue executing the rest of the script.
   that's important in the case if `.wuf/components.json` doesn't exists for example, freshly cloned repo.
   `rm` would throw and the rest of `update-components.sh` would be prevented from executing

Reply:
cool thanks